radixsort.py

```diff
@@ -0,0 +1,60 @@
+#! /usr/bin/env python
+import math
+import random
+import time
+
+
+def timed_func(func):
+    """Decorator for timing our insertion sort methods."""
+    def timed(*args, **kwargs):
+        start = time.time()
+        result = func(*args, **kwargs)
+        elapsed = time.time() - start
+        print "time expired: %s" % elapsed
+        return result
+    return timed
+
+
+@timed_func
+def radixsort(l):
+    """Implemented Radix Sort using math notation"""
+    try:
+        digits = int(math.log10(max(l))+1)
+        for i in xrange(1,digits+1):
+            ll = [[] for p in xrange(0,10)]
+            # import pdb; pdb.set_trace()
+            for x in l:
+                ll[x%10**i//(10**(i-1))].append(x)
```

Comment:
for readability's sake, you might consider binding a temporary symbol for the index you are building here.  This is pretty hard to parse, visualy.